### PR TITLE
Merging contents from https://github.com/vishalmmehra/microservices-datadriven-infra

### PR DESCRIPTION
**Pull request by vishalmmehra**
Merging contents from https://github.com/vishalmmehra/microservices-datadriven-infra

### CODE REVIEW

**Review by paulparkinson — COMMENTED** [review]
I think just "infra" might be a better dir name since the "microservices-datadriven" part would be redundant with the root dir but we can do that after the initial merge.

**Review by paulparkinson — APPROVED** [review]
We'll be making refactorings as discussed but approving so we can merge src in.

**Comment by renagranat** [comment]
My PR was approved.

**Pull request merged**
